src/tests/topp/DatabaseSuitability_out_1.tsv

```diff
@@ -1,7 +1,7 @@
 key	value
-#top_db_hits	438
+#top_db_hits	430

```

Comment:
Only 8 hits are filtered at a 1% FDR?

Reply:
Actually because the return calues `scoreHigherThanFDR_` are the wrong way around, there will only be 8 hits left. Which makes me want to add a seperate test for the FDR parameter..

Reply:
Hm yes or just change the FDR parameter for the tests.